API_Kraken/get_bitcon_price.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import datetime
+
+get_request = requests.get('https://api.kraken.com/0/public/Ticker?pair=XBTUSD')
+price = get_request.json()['result']['XXBTZUSD']['a'][0]
+date = datetime.datetime.now().strftime("%c")
+value = ('Price of bitcon for: ' + date + ' is: ' + price + ' USD')
+try:
+    print(value)
+except:
+    print("An error has occured")
+
+value_txt = {"Price of bitcon for:" : date, "is" : price}
+
+file = open("bitcoin_price.txt", "w")
+value_txt = repr(value_txt)
+file.write(value_txt + "USD")
```

Comment:
This wll produce entry in the file like:

`{'Price of bitcon for:': 'Mon Dec 18 18:35:20 2023', 'is': '41571.60000'}USD`

i think it should be simpler like:

`41571.60000,Mon Dec 18 18:35:20 2023`

Reply:
or if we want that to be in json format it should be like
```
[
  {'Price of bitcon for:': 'Mon Dec 18 18:35:20 2023', 'is': '41571.60000'}
]
```